pyne/mesh.py

```diff
@@ -974,6 +996,22 @@ def tag(self, name, value=None, tagtype=None, doc=None, size=None,
         dtype : numpy dtype, optional
             The data type of the tag. This only applies to NativeMeshTags. See PyMOAB
             for more details.
+        storage_type: str, optional
```

Comment:
I think the storage types listed below should be `'sparse'` and `'dense'` based on the type description here and the code below.